test/base/src/androidTest/kotlin/io/realm/test/shared/QueryTests.kt

```diff
@@ -903,29 +991,63 @@ class QueryTests {
     }
 
     @Test
-    @Ignore
     fun sum_asFlow() {
-        // TODO
+        for (propertyDescriptor in allPropertyDescriptorsForSum) {
+            aggregatorAssertions(AggregatorQueryType.SUM, propertyDescriptor)
+        }
     }
 
     @Test
-    @Ignore
     fun sum_asFlow_throwsIfInvalidType() {
-        // TODO
+        val value1 = 2
+        val value2 = 7
+
+        realm.writeBlocking {
+            copyToRealm(QuerySample(intField = value1))
+            copyToRealm(QuerySample(intField = value2))
+        }
+
+        runBlocking {
+            assertFailsWith<IllegalArgumentException> {
+                realm.query<QuerySample>()
+                    .sum<String>(QuerySample::intField.name)
+                    .asFlow()
+                    .collect { /* No-op */ }
+            }.let {
+                assertTrue(it.message!!.contains("Invalid numeric type"))
```

Comment:
Maybe we should somehow hint that it is the aggregators `T` that is of the invalid type. It could also be the actual field. Or is there more context in the full message?

Reply:
The exception message is
```
else -> throw IllegalArgumentException("Invalid numeric type for '$property', it is not a '${type.simpleName}'.")
```
But I will change it to `"Invalid numeric type for '$property', it is not a '${type.simpleName}' or cannot be represented by it."` to better reflect what is said in the documentation.